StableDiffusion-v1/cog_example/predict.py

```diff
@@ -48,7 +48,7 @@ def setup(self):
         print("Loading pipeline...")
 
         self.txt2img_pipe = StableDiffusionPipeline.from_pretrained(
-            "runwayml/stable-diffusion-v1-5",
+            "./stable-diffusion-v1-5",
```

Comment:
Does caching method need to change, just curious why/how this works without the full model name

Reply:
no it does not, this is an inaccurate modification I'd made for testing it out on runpod with a pytorch image, I'll have to fix this